explainer.md

```diff
@@ -46,207 +47,84 @@ This effort aims to create a JavaScript API
-
-In either case there's a one-to-one correspondence
-between a DOM node and a node in the accessibility tree,
-and there is minimal fine-grained control over the semantics of the corresponding accessibility node.
-
-## Gaps in the web platform's accessibility story
+(More background on existing APIs can be found in the [Appendices](#appendices).)
 
 Web apps that push the boundaries of what's possible on the web struggle to make them accessible
 because the APIs aren't yet sufficient -
 in particular, they are much less expressive than the native APIs that the browser communicates with.
 
-In the description of each of the phases of the Accessibility Object Model
-spec below, we discuss specific gaps -
-things that currently can't be achieved using current web accessibility APIs,
-or can't be achieved without cumbersome workarounds.
+1. Setting non-reflected (“default”) accessibility properties for [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) which can be overridden by page authors
+    - Currently, Web Components are forced to use ARIA to declare their default semantics.
+    This causes ARIA attributes which are really implementation details 
+    to "leak" into the DOM. 
+    - This capability _need not_, but _may_ be limited to Web Components.
+2.  Setting [relationship properties](https://www.w3.org/TR/wai-aria-1.1/#attrs_relationships) without needing to use IDREFs
+    - Currently, to specify any ARIA relationship,
+     an author must specify a unique ID on any element which may be the target
+     of the relationship.
+    - In the case of something like 
+    [`aria-activedescendant`](https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant),
+    this may be one of hundreds or thousands of elements, 
+    depending on the UI.
+    This requirement makes these APIs cumbersome to use
+    and lead to many extra DOM attributes being necessary.
+3. Listening for events from Assistive Technology
+   - Currently, _only_ built-in elements have the capability to react to events,
+     typically triggered by user actions such as 
+     ["click"](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_VIEW_CLICKED) 
+     or ["increment"](https://developer.apple.com/documentation/objectivec/nsobject/1615076-accessibilityincrement).
+4. Adding non-DOM nodes (“virtual nodes”) to the Accessibility tree 
+   - For example, to express a complex UI built out of a `<canvas>` element,
+     or streaming a remote desktop to a `<video>` element, etc.
+   - These should be able to express at least the same set of accessible properties as Elements, 
+     as well as parent/child/other relationships with other virtual nodes, 
+     and position/dimensions.
+5.  Introspecting the computed tree
+   - Developers currently have no way to probe or test how ARIA and other accessible properties are applied.
```

Comment:
This bullet needs to be indented in one more level to match the others.

Reply:
The problem was the second space above. Fixed.